### PR TITLE
remove ImageNet-C stimuli identifier to disable caching activations

### PR DESCRIPTION
**Pull request by mschrimpf**
* the activations are not used a second time, like for the ImageNet benchmark. so we would only store unnecessary information
* caching for temporal models like CORnet-S breaks when checking that coordinates are present because `logits-t0` doesn't match `logits`

### CODE REVIEW

**Comment by mschrimpf** [comment]
test failures primarily from S3 outage
